src/cloudant/result.py

```diff
@@ -359,6 +359,14 @@ def _parse_data(self, data):
         """
         return data.get('rows', [])
 
+    def all(self):
+        """
+        Retrieve the full results data set.
+
+        :return: results data set.
```

Comment:
Is it really a `set` as in the [built-in](https://docs.python.org/2/library/stdtypes.html#set)? We seem to refer to it as a collection elsewhere.